i18n/en.json

```diff
@@ -4,7 +4,9 @@
 			"Ævar Arnfjörð Bjarmason\r */\r"
 		]
 	},
-	"purge": "Purge",
+	"purge": "Purge cache",
 	"purge-descriptionmsg": "Adds a purge tab on all normal pages allowing for quick purging of the cache",
-	"purge-failed": "Purge failed"
+	"purge-failed": "Cache Purge failed",
+	"tooltip-n-purge-ext": "Purge cache of the current page",
+	"accesskey-n-purge-ext": "~"
```

Comment:
Why have `-n-` in these messages?

Reply:
Because that’s the appropriate message key for tooltip and quick access key. -n- is needed.

Reply:
The `-n-` in other messages means 'navigation' and is used for items e.g. in the sidebar. If any shorthand should be used, it'd be `-ca-`, but really there's no need outside of core.

Reply:
Hmm that’s weird I said n but added another comment right after mentioning -ca- then pushed a change making them ca. 

I see in some extensions not used, but in others it is.

Reply:
ah no worries, `-ca-` is good :)